src/libs/E2E/tests/linkingTest.e2e.ts

```diff
@@ -61,16 +59,7 @@ const test = (config: NativeConfig) => {
                 return;
             }
 
-            if (entry.name === CONST.TIMING.REPORT_INITIAL_RENDER) {
-                console.debug('[E2E] Navigating to linked report action…');
-                Timing.start(CONST.TIMING.SWITCH_REPORT);
-                Performance.markStart(CONST.TIMING.SWITCH_REPORT);
-
-                Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(linkedReportID, linkedReportActionID));
```

Comment:
I imagine not navigating here breaks the flow

Reply:
Yes

Reply:
oki doki, solving this in another discussion so @perunt can follow on this very test